docs/connectors/sqlserver.asciidoc

```diff
@@ -76,6 +76,17 @@ EXEC sys.sp_cdc_enable_table
 GO
 ----
 
+Verify that the user have access to the cdc table
+[source, sql]
+----
+-- =========
+-- Verify the user of the connector have access, this query should not have empty result
```

Comment:
Nice improvement!

@jpechane, could we somehow check this programmatically during connector start-up?

Reply:
Already logged - https://issues.jboss.org/browse/DBZ-1281

Reply:
Ok, sweet. Merging this one here.